src/lib/browse/colors.ts

```diff
@@ -34,9 +34,8 @@ export const colors = {
     "#080C54",
   ],
 
-  atf2: "#00AFFF",
-  atf3: "#FF62DC",
-  atf4: "#FFD833",
+  // A qualitative set from colorbrewer2.org
+  funding_programmes: ["#66c2a5", "#fc8d62", "#8da0cb", "#e78ac3"],
```

Comment:
Actually we may want 5 (When we've resolved data issues):
ATF2, ATF3, ATF4, ATF4e, LCWIP.

No?

Reply:
To make the final category broader, we could also say "Planned pipeline" so it's not tied to any particular process or timescale.

Reply:
Pete: Fixed, changed to 6 colors for more future-proofness.
Robin: Good point, https://github.com/acteng/atip-scheme-data/pull/11